src/validate-and-gen-trees.sh

```diff
@@ -98,7 +98,7 @@ rm ../bin/*-rib-tree.txt.tmp
 
 echo "Validating examples"
 
-for i in yang/example-bgp-configuration-*.xml
+for i in yang/example-bgp-configuration-a.*.xml
```

Comment:
I suggest that changes to the code outside the scope of the issue get fixed under their own issue.

Reply:
I would love to, and understand the sentiment behind keeping the commit limited to the immediate issue at hand.

Except without this change, I am getting compile errors and the draft does not build and to fix all of that requires me to do even more surgery than I would like to.

Reply:
Understood.  It wasn't an obvious place where the change would result in errors.  I did try it out and don't think I noticed anything.  I must have missed it.